Datadog/IntegrationUnitTests/Public/TelemetryCoreIntegrationTests.swift

```diff
@@ -60,6 +60,10 @@ class CoreTelemetryIntegrationTests: XCTestCase {
         config.metricsTelemetrySampleRate = 100
         RUM.enable(with: config, in: core)
 
+        // When
+        RUMMonitor.shared(in: core).startView(key: "View")
+        RUMMonitor.shared(in: core).stopView(key: "View")
```

Comment:
The test is:
```swift
func testGivenRUMEnabled_whenNoViewIsActive_telemetryEventsAreLinkedToSession()
```
so the "when" condition should be "no view is active". Because we start `ApplicationLaunch` view with `RUM.enable()` it makes an active view. To make this test pass I had to force "no active view" situation, which can be only done by starting and stopping another view (think: we can't stop `ApplicationLaunch` view).

I'll fix `// Given & When`, good notice 👌